### PR TITLE
Moved Color.getRaylibColor() to RaylibUtil.getRaylibColor()

### PR DESCRIPTION
**Pull request by andreaevolo**

### CODE REVIEW

**Comment by Yeffian** [comment]
Thanks for contributing to Orbital! I will review and merge this pull request in a few hours, bit busy right now :D

**Review by Yeffian — CHANGES_REQUESTED** [review]
Looks good, but you haven't changed the uses of the class version of the function to the RaylibUtil version.

**Comment by andreaevolo** [comment]
@YeffyCodeGit My bad. I'll fix it right now.

**Review by Yeffian — CHANGES_REQUESTED** [review]
The function should take in a parameter for the `Color` to turn into a Raylib color, your current code would not compile as the r, g, b and a variables are undefined. Please make sure your code code actually compiles before making a push.

**Comment by andreaevolo** [comment]
javara
Cattujaavaa
That was the problem right? ( I'll fix all the other methods too in the Randerer class ). @YeffyCodeGit

**Comment by andreaevolo** [comment]
Cat1
Cat2
Is this the problem you were talking about? If yes I'll fix all the methods inside the renderer class.

But then I need to fix this. 
Cattura4

File: src/main/java/dev/yeff/orbital/graphics/Renderer.java [hunk 1]
```diff
@@ -63,7 +63,7 @@ public static void drawTexture(Sprite sprite, Vector2f pos) {
      * @param radius The radius of the circle.
      */
     public static void drawCircle(Color color, Vector2f pos, float radius) {
-        DrawCircleV(RaylibUtil.getAsRaylibVec2(pos), radius, RaylibUtil.getRaylibColor());
+        DrawCircleV(RaylibUtil.getAsRaylibVec2(pos), radius, RaylibUtil.getRaylibColor(color));
```
Comment (Yeffian) [inline src/main/java/dev/yeff/orbital/graphics/Renderer.java:66]: The function definition does not take a `Color` object, so this won't compile again. Please make sure the code your submitting is correct.

Again, the function needs to take a `Color` parameter, and the r, g, b and a values need to come from the parameter.

**Review by Yeffian — CHANGES_REQUESTED** [review]

**Comment by andreaevolo** [comment]
@YeffyCodeGit I messed up something with git because I did change the function definition but my changes weren't reflected in the commit. Sorry for the inconvenient.

**Comment by Yeffian** [comment]
In that case, could you push the changes again?

**Comment by andreaevolo** [comment]
Cattura
that's how I changed the method definition in my local code.

**Comment by andreaevolo** [comment]
@YeffyCodeGit Yeah I'll check what went wrong and send a push again.

**Comment by Yeffian** [comment]
LGTM!

**Pull request merged**